client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,36 @@
+import { Grid, Paper } from '@material-ui/core';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import useStyles from './useStyles';
+import { Route } from 'react-router-dom';
+import ProfileMenu from '../../components/ProfileMenu/ProfileMenu';
+import ProfileEdit from './ProfileEdit/ProfileEdit';
+
+export default function Profile(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Grid className={`${classes.root}`}>
+      <Grid container spacing={2} className={`${classes.profile}`}>
+        <CssBaseline />
```

Comment:
The `CssBaseline' is a global reset component, ideally it should be included only once in the App.tsx

Ref: https://mui.com/components/css-baseline/